### PR TITLE
[DEVOPS-528]  installer | OS X:  convert to the Buildkite bucket

### PR DESCRIPTION
**Pull request by deepfire**
Switch to Buildkite-provided Cardano builds.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -9,7 +9,7 @@ env:
   global:
     - VERSION=1.0
     - NETWORK=mainnet
-    - CARDANO_SL_BRANCH=cardano-sl-1.0
+    - CARDANO_SL_BRANCH=devops-497-kite
```
Comment (domenkozar) [inline .travis.yml]: Should be master

Comment (deepfire) [reply .travis.yml]: Done

**Comment by domenkozar** [comment]
Note that this can't be merged until buildkite lands in cardano-sl master.

**Comment by deepfire** [comment]
@domenkozar, CI has passed..

File: .travis.yml [hunk 2]
```diff
@@ -9,7 +9,7 @@ env:
   global:
     - VERSION=1.0
     - NETWORK=mainnet
-    - CARDANO_SL_BRANCH=cardano-sl-1.0
+    - CARDANO_SL_BRANCH=release/1.0.4
```
Comment (domenkozar) [inline .travis.yml]: This should be `master`.

Comment (domenkozar) [reply .travis.yml]: Also it should always be in sync with appveyor

Comment (deepfire) [reply .travis.yml]: Done..

**Comment by domenkozar** [comment]
Squashed and pushed, now that master of cardano-sl is using buildkite, CI should pass here.

**Comment by domenkozar** [comment]
This requires a backport to 0.8.2 as well.

**Comment by domenkozar** [comment]
0.8.2: https://github.com/input-output-hk/daedalus/pull/648

**Pull request merged**